package.json

```diff
@@ -26,34 +26,32 @@
     },
     "homepage": "https://github.com/fluid-project/uio-plus#ui-options-plus-uio",
     "scripts": {
-        "lint": "grunt lint",
-        "pretest": "node node_modules/rimraf/bin.js reports/* coverage/* && grunt build",
-        "test": "npm run test:node && npm run test:browser",
-        "test:node": "node node_modules/nyc/bin/nyc.js node tests/node/all-tests.js",
-        "test:browser": "node node_modules/testem/testem.js ci --file tests/browser/testem.js",
-        "test:vagrant": "vagrant up && vagrant ssh -c 'cd /home/vagrant/sync/; DISPLAY=:0 npm test'",
-        "test:vagrantBrowser": "vagrant up && vagrant ssh -c 'cd /home/vagrant/sync/; DISPLAY=:0 npm run test:browser'",
-        "test:vagrantNode": "vagrant up && vagrant ssh -c 'cd /home/vagrant/sync/; DISPLAY=:0 npm run test:node'",
-        "posttest": "node node_modules/nyc/bin/nyc.js report --reporter text-summary --reporter html"
+        "lint": "fluid-lint-all",
+        "pretest": "run-s -l clean:test",
+        "test": "run-s test:node test:browser",
+        "test:node": "nyc node tests/node/all-tests.js",
+        "test:browser": "testem ci --file tests/browser/testem.js",
+        "test:report": "nyc report -r text-summary -r html -r json",
+        "posttest": "npm run test:report",
+        "clean": "run-p clean:*",
+        "clean:build": "rimraf dist",
+        "clean:test": "rimraf reports coverage",
+        "build": "run-s -l clean:build build:*",
+        "build:dist": "cpy . ../dist --cwd=src && cpy . ../../dist/lib/infusion --cwd=node_modules/infusion"
     },
     "dependencies": {
-        "infusion": "3.0.0-dev.20200525T161532Z.fc4d4b3.FLUID-6145"
+        "infusion": "4.6.0"
     },
     "devDependencies": {
-        "eslint-config-fluid": "1.4.0",
-        "gpii-grunt-lint-all": "1.0.7",
+        "cpy-cli": "4.2.0",
+        "eslint-config-fluid": "2.1.1",
+        "fluid-lint-all": "1.2.3",
         "fluid-testem": "2.1.15",
-        "grunt": "1.5.2",
-        "grunt-cli": "1.4.3",
-        "grunt-contrib-clean": "2.0.0",
-        "grunt-contrib-copy": "1.0.0",
-        "grunt-contrib-stylus": "1.2.0",
-        "grunt-shell": "4.0.0",
         "node-jqunit": "1.1.9",
+        "npm-run-all": "4.1.5",
         "nyc": "15.1.0",
         "rimraf": "3.0.2",
-        "sinon": "13.0.1",
```

Comment:
I'm getting an error when trying to run the tests that it can't find sinon.

Reply:
Interesting. `sinon-chrome` pulls in `sinon` on my computer. I've added the `sinon`. Can you try again? Thanks.